src/DIRAC/Core/Base/__init__.py

```diff
@@ -1,2 +1,5 @@
-""" For backward compatability """
+"""For backward compatability.
+This allows users continue to use old stype `from DIRAC.Core.Base import Script`,
+but normally developer should use `from DIRAC.Core.Base.Script import Script`
+"""
 from DIRAC.Core.Base.Script import Script
```

Comment:
> I still don't see why this is needed?

https://github.com/DIRACGrid/DIRAC/pull/5713#issuecomment-1028914090 some discussion arose, the essence of which (at least as I understood it, maybe @andresailer will correct me) is that there are DIRAC users who have long used `from DIRAC.Core.Base import Script`, etc. in their personal codes and any changes in this, namely the use of `from DIRAC.Core.Base.Script import Script` complicate their lives, which we want to avoid.

Personally I would label it deprecated. 

> It also causes these two seemingly identical objects to be different

as it is for backward compatibility, instead of as the recommended way I do not see in it a problem, in both cases actions are conducted on the same class:
```python
>>> from DIRAC.Core.Base import Script
>>> from DIRAC.Core.Base.Script import parseCommandLine  
>>> Script.localCfg.isParsed
False
>>> parseCommandLine()
No proxy found
([], [])
>>> Script.localCfg.isParsed
True
```

Reply:
Ah sorry, I misunderstood your comment.. I deleted this import, now its OK
```python
>>> from DIRAC.Core.Base import Script
>>> from DIRAC.Core.Base.Script import parseCommandLine
>>> Script.parseCommandLine == parseCommandLine
True
```